command/ca/provisioner/caConfigClient.go

```diff
@@ -19,62 +19,66 @@ func newNoDB() *nodb {
 	return &nodb{}
 }
 
-func (n *nodb) CreateProvisioner(ctx context.Context, prov *linkedca.Provisioner) error {
+func (n *nodb) CreateProvisioner(context.Context, *linkedca.Provisioner) error {
 	return nil
 }
 
-func (n *nodb) GetProvisioner(ctx context.Context, id string) (*linkedca.Provisioner, error) {
+func (n *nodb) GetProvisioner(_ context.Context, id string) (*linkedca.Provisioner, error) {
+	_ = id
 	//nolint:nilnil // nodb is a noop interface.
 	return nil, nil
 }
 
-func (n *nodb) GetProvisioners(ctx context.Context) ([]*linkedca.Provisioner, error) {
+func (n *nodb) GetProvisioners(context.Context) ([]*linkedca.Provisioner, error) {
 	return nil, nil
 }
 
-func (n *nodb) UpdateProvisioner(ctx context.Context, prov *linkedca.Provisioner) error {
+func (n *nodb) UpdateProvisioner(context.Context, *linkedca.Provisioner) error {
 	return nil
 }
 
-func (n *nodb) DeleteProvisioner(ctx context.Context, id string) error {
+func (n *nodb) DeleteProvisioner(_ context.Context, id string) error {
+	_ = id
```

Comment:
This might make more sense, it's a noop db, there no need to document it.
```suggestion
func (n *nodb) DeleteProvisioner(context.Context, string) error {
```